packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -381,7 +381,7 @@ export class FluidDevtools implements IFluidDevtools {
 	private getSupportedFeatures(): DevtoolsFeatureFlags {
 		return {
 			telemetry: this.logger !== undefined,
-			// Completed but disabled until we finish the story for unsampled telemetry that powers this feature
+			// Most work completed, but not ready to completely enable.
```

Comment:
Now that we have the documentation for how to enable unsampled telemetry in the op latency view itself, I think we can remove this comment and flip the flag to `true`. Can you double check with @rohandubal ?